### DISCUSSION TITLE
[@types/react-alert] Missing declaration for removeAll

### DISCUSSION THREAD

**Discussion by lucas-gauer** [DefinitelyTyped/DefinitelyTyped#58908] [Issues with a @types package]
React-alert v7 has a method to `removeAll` the alerts. Currently is not listed on the type definitions.
I understand that the current support is for version 5.2 but version 7 has been out for 2 years+

[a comment by an automated account is not shown]

**Comment by g1eny0ung** [comment]
@lucas-gauer There is indeed no update for a long time, I will update it in my free time. 😃

**Reply by g1eny0ung** [reply]
Updated in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59016.

**Reply by g1eny0ung** [reply]
@Lucas-gauer I rewrote the whole `react-alert` and divided the version according to the past releases. If you have some free time you can help to see if there are any problems (You can comment on the PR). Thanks a lot. 😃
